src/common/tusb_fifo.h

```diff
@@ -134,6 +160,18 @@ static inline uint16_t tu_fifo_depth(tu_fifo_t* f)
   return f->depth;
 }
 
+// When writing into the FIFO by fifo_write_n(), rd_mode determines how the pointer read from is modified
+static inline void tu_fifo_set_copy_mode_read(tu_fifo_t* f, tu_fifo_copy_mode_t rd_mode)
```

Comment:
instead of setting copy_mode, I think it would be better to have an dedicated API for read_n() write_n() for this mode e.g
`tu_fifo_read_n_const_addr()`, `tu_fifo_write_n_const_addr()` ( you can name it differently). That way we don't have to mange the mode internally. It also make the usage more explicitly (const address) where it is called.

Reply:
Very true, i will change that!